pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -0,0 +1,142 @@
+package certificate
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestCertManager(t *testing.T) {
+	test.NewTest(t).Id("T38").Groups(test.Full, test.ARM, test.InterOp).Run(func(t test.TestHelper) {
+
+		ns := "foo"
+		cert := "cert-manager"
+		t.Cleanup(func() {
+			oc.DeleteNamespace(t, cert)
+			oc.RecreateNamespace(t, ns)
+		})
+
+		t.LogStep("uninstall the SMCP")
+		oc.RecreateNamespace(t, meshNamespace)
+
+		t.LogStep("Add jetstack repo to helm")
+		shell.Execute(t, `helm repo add jetstack https://charts.jetstack.io`)
+
+		t.LogStep("Install cert-manager")
+		shell.Execute(t,
+			fmt.Sprintf(`helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.11.0 --set installCRDs=true`),
+			assert.OutputContains("cert-manager v1.11.0 has been deployed successfully",
+				"Successfully installed cert-manager",
+				"Failed to installed cert-manager"))
+
+		t.LogStep("Provision certificates")
+		oc.ApplyString(t, meshNamespace, SelfSignedCa)
+
+		t.LogStep("Install cert-manager istio-csr Service")
+		shell.Execute(t,
+			fmt.Sprintf(`helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f https://raw.githubusercontent.com/maistra/istio-operator/maistra-2.4/deploy/examples/cert-manager/istio-csr-helm-values.yaml`),
+			assert.OutputContains("STATUS: deployed",
+				"Successfully, installed cert-manager-istio-csr",
+				"Failed to installed cert-manager-istio-csr"))
+
+		t.LogStep("deploy the cert-manager in SMCP")
+		oc.ApplyString(t, meshNamespace, CertManagerSMCP)
+		oc.WaitSMCPReady(t, meshNamespace, smcpName)
+
+		t.LogStep("Install httpbin and sleep")
+		app.InstallAndWaitReady(t, app.Httpbin(ns), app.Sleep(ns))
+
+		t.LogStep("Check if httpbin returns 200 OK ")
+		retry.UntilSuccess(t, func(t test.TestHelper) {
+			oc.Exec(t,
+				pod.MatchingSelector("app=sleep", ns),
+				"sleep",
+				fmt.Sprintf(`curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%%%%{http_code}\n"`),
```

Comment:
I fixed the isse with the percent signs, so you no longer need to escape it twice.
```suggestion
				fmt.Sprintf(`curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%%{http_code}\n"`),
```

Reply:
updated with latest commit.